generator/uscore/uscore_generator.rb

```diff
@@ -138,7 +140,8 @@ def create_include_test(sequence)
         include_test = {
           tests_that: "Server returns the appropriate resource from the following _includes: #{sequence[:include_params].join(', ')}",
           index: sequence[:tests].length + 1,
-          link: 'https://www.hl7.org/fhir/search.html#include'
+          link: 'https://www.hl7.org/fhir/search.html#include',
+          description: "A Server SHOULD be capable of supporting the following _includes: #{sequence[:include_params].join(', ')}"
```

Comment:
This says `SHOULD`, but the test is not optional.

Reply:
Fixed